heuristic/repair_operators.py

```diff
@@ -25,32 +25,38 @@ def worst_week_repair(shifts_in_week, competencies, t_covered_by_shift, employee
     while(True):
         calculate_deviation_from_demand(state, competencies, t_covered_by_shift, employee_with_competencies, demand, changed)
         delta_calculate_negative_deviation_from_contracted_hours(state, employees_changed, contracted_hours, weeks, time_periods_in_week, competencies, time_step)
-        shifts =    {
-                    (t1, v1, competencies): -sum(state.soft_vars["deviation_from_ideal_demand"][c,t] for c in competencies for t in t_covered_by_shift[t1, v1] if (c,t) in state.soft_vars["deviation_from_ideal_demand"])
-                                            - (20*(len(competencies)-1) + v1) for competencies in employee_with_competency_combination for t1, v1 in shifts_in_week[week[0]] if (t1,v1,competencies) not in impossible_shifts
-                    }
+
+        shifts = {
+                (t1, v1, competencies): -sum(
+                    state.soft_vars["deviation_from_ideal_demand"][c,t]
+                    for c in competencies
+                    for t in t_covered_by_shift[t1, v1] if (c,t) in state.soft_vars["deviation_from_ideal_demand"])
+                    - (20*(len(competencies)-1) + v1)
+                    for competencies in employee_with_competency_combination
+                    for t1, v1 in shifts_in_week[week[0]] if (t1,v1,competencies) not in impossible_shifts
```

Comment:
This causes: 
```python
TypeError: 'int' object is not iterable
```

Any idea why @hkgrov ?

Reply:
Not at the moment, no. I don't have the issue in my branch. Can look into it.